api/v1/storagecluster_types.go

```diff
@@ -429,6 +429,15 @@ type MultiCloudGatewaySpec struct {
 	// for nooba-db pods
 	// +optional
 	DbStorageClassName string `json:"dbStorageClassName,omitempty"`
+
+	// DBBackup (optional) configure automatic scheduled backups of noobaa database volume.
+	// +optional
+	DBBackup *nbv1.DBBackupSpec `json:"dbBackup,omitempty"`
+
+	// DBRecovery (optional) configure database recovery from snapshot
+	// +optional
+	DBRecovery *nbv1.DBRecoverySpec `json:"dbRecovery,omitempty"`
```

Comment:
to be inline with the existing fields abbrv. (`DbStorageClassName`), better these new fields also be `DbBackup` and `DbRecovery`? just an observation.